src/dotnet/APIView/ClientSPA/src/app/_components/review-nav/review-nav.component.scss

```diff
@@ -42,4 +42,8 @@
     .p-tree-container {
         padding-left: 0px !important;
     }
+
+    .client-type {
+        font-weight: bold !important;
```

Comment:
This did not add much visual difference.

![ClientAPI](https://github.com/user-attachments/assets/1c4e8610-5ca2-4d63-95f8-6c2c563fc70b)

Reply:
addressed with more prominent changes